ccd-definition/CaseField/CareSupervision/CMO-prod.json

```diff
@@ -1,59 +0,0 @@
-[
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "ID": "pastHearingsForCMO",
-    "Label": "Which CMO do you want to send to the judge to review?",
-    "HintText": "All CMOs must have been discussed by all parties at the hearing.",
-    "FieldType": "DynamicList",
-    "SecurityClassification": "Public"
-  },
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "ID": "numHearingsWithoutCMO",
-    "Label": "Number of past hearings without a cmo",
-    "FieldType": "Text",
-    "SecurityClassification": "Public"
-  },
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "ID": "noHearingsForCMOs",
-    "Label": "You cannot upload an agreed CMO.\n\nThis is because the judge is considering, or has sealed, the CMOs from all past hearings.",
-    "FieldType": "Label",
-    "SecurityClassification": "Public"
-  },
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "ID": "multiHearingsWithCMOs",
-    "Label": "Hearings with CMOs already being reviewed",
-    "FieldType": "TextArea",
-    "SecurityClassification": "Public"
-  },
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "ID": "singleHearingWithCMO",
-    "Label": "Hearings with CMOs already being reviewed",
-    "FieldType": "TextArea",
-    "SecurityClassification": "Public"
-  },
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "ID": "showHearingsMultiTextArea",
-    "Label": "Field to store conditional result for showing multiHearingsWithCMOs",
-    "FieldType": "Text",
-    "SecurityClassification": "Public"
-  },
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "ID": "showHearingsSingleTextArea",
```

Comment:
is it safe to delete these fields completely? If some prod case already contains any of this filed we will get exception from ccd that no definition present

Reply:
this is one of the transient fields 
    public static String[] transientFields() {
        return new String[]{
            "showCMOsSentToJudge", "cmosSentToJudge", "cmoUploadType", "pastHearingsForCMO", "futureHearingsForCMO",
            "cmoHearingInfo", "showReplacementCMO", "previousCMO", "uploadedCaseManagementOrder", "replacementCMO",
            "cmoSupportingDocs", "cmoJudgeInfo", "cmoToSend",
            // TODO: 19/10/2020 Delete after this comment when FPLA-2019 is toggled on
            "numHearingsWithoutCMO", "singleHearingWithCMO", "multiHearingsWithCMOs", "showHearingsSingleTextArea",
            "showHearingsMultiTextArea"
        };
    }
@tomaszpowroznik 
do you reckon that there's a high chance to get that when the event is in flight?

Reply:
Just added CMO-deprecated with the transient deprecated fields

Reply:
if they are temporary thats fine, but just checked and noHearingsForCMOs is not removed in controller